modin/core/execution/ray/common/utils.py

```diff
@@ -74,6 +74,10 @@ def initialize_ray(
     """
     extra_init_kw = {"runtime_env": {"env_vars": {"__MODIN_AUTOIMPORT_PANDAS__": "1"}}}
     if not ray.is_initialized() or override_is_cluster:
+        # TODO(https://github.com/ray-project/ray/issues/28216): remove this
+        # workaround once Ray gives a better way to suppress task errors.
+        # Ideally we would not set global environment variables.
+        os.environ["RAY_IGNORE_UNHANDLED_ERRORS"] = "1"
```

Comment:
Maybe first look to see if the variable is defined by the user, and if not, then define it here?

Reply:
Done.